consensus/ibft/signer/extra.go

```diff
@@ -37,6 +42,23 @@ type Seals interface {
 	UnmarshalRLPFrom(*fastrlp.Parser, *fastrlp.Value) error
 }
 
+func parseRoundNumber(v *fastrlp.Value) (*uint64, error) {
```

Comment:
Can we use  `num := binary.LittleEndian.Uint64(array)` or `num := binary.BigEndian.Uint64(array)` here?

Reply:
Not it uses GetUint64, fastrlp provides originally